lib/dirac.cpp

```diff
@@ -210,18 +210,16 @@ namespace quda {
       return new DiracImprovedStaggeredKD(param);
     } else if (param.type == QUDA_TWISTED_CLOVER_DIRAC) {
       if (getVerbosity() >= QUDA_DEBUG_VERBOSE) printfQuda("Creating a DiracTwistedClover operator (%d flavor(s))\n", param.Ls);
-      if (param.Ls == 1) {
-	return new DiracTwistedClover(param, 4);
-      } else { 
-	errorQuda("Cannot create DiracTwistedClover operator for %d flavors\n", param.Ls);
-      }
+      if (param.Ls == 1)
+        return new DiracTwistedClover(param, 4);
+      else
+        return new DiracTwistedClover(param, 5);
```

Comment:
Let's reintroduce a `errorQuda` path for `param.Ls != 1` and `2`, just to be safe.

Reply:
e0185c7fefda80fac0284473e25ee4d9a010b7d9